### ISSUE TITLE
Provide option to get all instances of the active slide

### ISSUE THREAD

**Issue by julmot** [nolimits4web/Swiper#1485]
I have built a custom slideshow with overlay elements that will slide in with a specific delay to the swiper. So for example I'm hooking into `onSlideChangeStart` and wait a short delay and show my overlay elements. However, I also use the loop-mode that will clone elements. If I want to show my overlays I need to get all instances of the current slide, because otherwise I may only show the overlay in one instance and if I swipe to the next slide the slide will be cloned and the overlay will be hidden. Because `swiper.activeIndex` only returns the current slide and there is no way to find out the slide of the cloned one, I need to pass around this with custom `data`-attributes.

In summary there should be an option to get all instances of the active slide to make changes to all of them!

Workaround code:

``` javascript
// Because siblings will be added dynamically
// before and after because of the loop functionality,
// we can not detect the active element by it's index.
// It will be cloned and used multiple times, so we need
// to find it's custom index "data-slide" and find all
// occurences of this slide with it and make the changes to all
// of them.
var attr = $(".swiper-slide:nth-child(" + (swiper.activeIndex + 1) + ")").attr("data-slide");
var $active = $(".swiper-slide[data-slide='" + attr + "']");
```

**Comment by nolimits4web** [comment]
Sorry, is it too hard to use these 2 simple lines?

**Comment by julmot** [comment]
The plugin offers tons of options but isn't able to just offer this simple parameter? Is it too hard to pass a reference to all active elements?

**Comment by nolimits4web** [comment]
> isn't able to just offer this simple parameter

Because simple things should be done in simple ways, without overloading lib itself with extra code which can be done in one line. The same as why there is no methods to add/remove classes/attributes to/from slides, get slide index, etc. Because they are easily achieved with 1-2 lines.

Anyway, maybe will add something for that in next release

**Comment by danielocdh** [comment]
How do I solve this if I'm using 6 slides, I want to use the loop option and there is a lot of user interaction/dom changes on some of the slides sometimes?

**Comment by nolimits4web** [comment]
Issue is closed because of outdated/irrelevant/not actual/needed more information/inactivity.

If this issue is still actual and reproducible for latest version of Swiper, please create new issue and fill the issue template correctly:
* Clearly describe the issue including steps to reproduce when it is a bug.
* Make sure you fill in the earliest version that you know has the issue.
* Provide live link or JSFiddle/Codepen or website with isssue
